index.html

```diff
@@ -184,7 +184,7 @@ <h3>SAMI</h3>
           <div class="col-md-4">
           <h3>SCAP SECURITY GUIDE (SSG)</h3>
           <p>Delivers security guidance, baselines, and associated validation mechanisms using the Security Content Automation Protocol (SCAP) for hardening Red Hat products.<br></p>
-          <a class="btn btn-primary" href="https://fedorahosted.org/scap-security-guide" target="_blank" role="button">View Repo Page</a> <!-- <a class="learn-more" target="_blank" href="#">Learn More <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span></a> -->
+          <a class="btn btn-primary" href="https://github.com/openscap/scap-security-guide" target="_blank" role="button">View Repo Page</a> <!-- <a class="learn-more" target="_blank" href="#">Learn More <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span></a> -->
```

Comment:
This redirects to https://github.com/ComplianceAsCode/content
Is that the new location?

Or is the following the new canonical location for this content? (this link appears at the github repo)
https://www.open-scap.org/security-policies/scap-security-guide/

Reply:
> This redirects to https://github.com/ComplianceAsCode/content
> Is that the new location?

@ctubbsii it is the newest location. I kept the legacy `github.com/openscap/scap-security-guide` since some people familiar with the project might still prefer that. Would you prefer the `https://github.com/ComplianceAsCode/content` instead?

Reply:
I personally think it's less confusing to use the current location, rather than the legacy one. I just wanted to make sure that redirect was intentional. As for the open-scap.org link, I guess I wasn't paying close enough attention in my previous comment that the link being updated is the "Repo Page" link. My mistake :smiley_cat:

Reply:
> I personally think it's less confusing to use the current location, rather than the legacy one. I just wanted to make sure that redirect was intentional. 

@ctubbsii cool. I have updated the repo link.

> As for the open-scap.org link, I guess I wasn't paying close enough attention in my previous comment that the link being updated is the "Repo Page" link. My mistake smiley_cat

No worries at all. :)